### DISCUSSION TITLE
xcode / commandline tools question

### DISCUSSION THREAD

**Discussion by Oclair** [Homebrew/discussions#365] [Getting started]
Hi everyone,
I saw this today
```
Warning: Your Xcode (12.2) is outdated.
Please update to Xcode 12.3 (or delete it).
Xcode can be updated from the App Store.
```
Does this mean I can simply delete Xcode and install the latest version of the command line tools without it?
I'd love to NOT have Xcode taking all that precious space on my 2018MBP

thanks in advance
Oclair

**Comment by SMillerDev** [comment]
Most formula work fine without xcode

**Reply by OldSubSailor** [reply]
That means yes, you can delete Xcode, but you should to go through the command line tools reinstallation process again which is simple enough. Removing Xcode seemed to mess up my installed command line tools and gave fake installation verification results. I reinstalled and things were fixed.
So from terminal just go through the process again. 
xcode-select --install
